src/Tests/API/APIApprovals.ApproveAzureServiceBusTransport.approved.txt

```diff
@@ -65,8 +65,7 @@ namespace NServiceBus
         public NServiceBus.AzureServiceBusMessageSenderSettings BackOffTimeOnThrottle(System.TimeSpan backoffTime) { }
         public NServiceBus.AzureServiceBusMessageSenderSettings MaximuMessageSizeInKilobytes(int sizeInKilobytes) { }
         public NServiceBus.AzureServiceBusMessageSenderSettings MessageSizePaddingPercentage(int percentage) { }
-        public NServiceBus.AzureServiceBusMessageSenderSettings OversizedBrokeredMessageHandler<T>(T instance)
```

Comment:
Discussed this with @yvesgoelevan and we figured it is OK to do this in a major. There is no way we can deprecate this API properly that's why it was directly replaced